app/assets/stylesheets/static.scss

```diff
@@ -0,0 +1,20 @@
+.site {
+  background: $orange;
+  padding-top: 20px;
+
+  .container {
```

Comment:
although nesting selectors is not bad it's better when you don't do it since this generates more specificity and it's a problem to override.